cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -141,7 +141,11 @@ object EvaluatorCommand : CommandWithHelp() {
 
         if (absoluteOutputDir != null) {
             // Note: This overwrites any existing EvaluatorRun from the input file.
-            val ortResultOutput = ortResultInput.copy(evaluator = evaluatorRun)
+            val ortResultOutput = ortResultInput.copy(evaluator = evaluatorRun).apply {
+                ortResultInput.data.forEach { key, value ->
+                    data[key] = value
+                }
```

Comment:
You could use `data += ortResultInput.data` instead.

Reply:
Thanks, I could have sworn I tried that but was running into problems with `data` being of `Any` type, but now it works.